### PR TITLE
Add keds spacing

### PR DESCRIPTION
**Pull request by jmagalhaes-xing**

### CODE REVIEW

File: src/app/core/_grids.scss [hunk 1]
```diff
@@ -0,0 +1,11 @@
+@mixin for-screen($size) {
```
Comment (RicardoNeves) [inline src/app/core/_grids.scss]: how do we override media queries?
we should also add the keds prefix here.

File: src/app/spacing/shared/directives/spacing-bottom/spacing-bottom.stories.ts [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import { moduleMetadata, storiesOf } from '@storybook/angular';
+
+import { SpacingModule } from '../../../spacing.module';
+
+storiesOf('Spacing Module/Spacing Bottom Directive', module)
+  .addDecorator(moduleMetadata({
+    imports: [SpacingModule],
+  }));
+
+storiesOf('Spacing Module/Spacing Bottom Directive', module)
```
Comment (RicardoNeves) [inline src/app/spacing/shared/directives/spacing-bottom/spacing-bottom.stories.ts]: /Sizes?

File: src/app/spacing/shared/directives/spacing-left/spacing-left.stories.ts [hunk 3]
```diff
@@ -0,0 +1,37 @@
+import { moduleMetadata, storiesOf } from '@storybook/angular';
+
+import { SpacingModule } from '../../../spacing.module';
+
+storiesOf('Spacing Module/Spacing Left Directive', module)
+  .addDecorator(moduleMetadata({
+    imports: [SpacingModule],
+  }));
+
+storiesOf('Spacing Module/Spacing Left Directive', module)
```
Comment (RicardoNeves) [inline src/app/spacing/shared/directives/spacing-left/spacing-left.stories.ts]: /sizes?

File: src/app/typography/shared/directives/text/text.directive.spec.ts [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+describe('TextDirective', () => {
+  let textDirective: TextDirective;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      providers: [
+        {
+          provide: ElementRef,
+          useValue: new ElementRef(document.createElement('span')),
```
Comment (RicardoNeves) [inline src/app/typography/shared/directives/text/text.directive.spec.ts]: why span this time?

File: src/app/spacing/shared/directives/spacing-bottom/spacing-bottom.stories.ts (hunk 2 above)
Comment (jmagalhaes-xing) [reply src/app/spacing/shared/directives/spacing-bottom/spacing-bottom.stories.ts]: Spacing is only sizes, I don't think there's a need for specification

File: src/app/spacing/shared/directives/spacing-left/spacing-left.stories.ts (hunk 3 above)
Comment (jmagalhaes-xing) [reply src/app/spacing/shared/directives/spacing-left/spacing-left.stories.ts]: Same

File: src/app/typography/shared/directives/text/text.directive.spec.ts (hunk 4 above)
Comment (jmagalhaes-xing) [reply src/app/typography/shared/directives/text/text.directive.spec.ts]: Why not? :man_shrugging:

**Comment by jmagalhaes-xing** [comment]
No. https://www.figma.com/file/znlNioi6HefdMsTLgsDTbX/KEDS%2FGrids_Spacings?node-id=0%3A283

**Comment by RicardoNeves** [comment]
> No. https://www.figma.com/file/znlNioi6HefdMsTLgsDTbX/KEDS%2FGrids_Spacings?node-id=0%3A283

Then remove it please. I ran storybook and we have the size there.

File: src/app/spacing/shared/directives/spacing-bottom/spacing-bottom.stories.ts [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+      spacing: 'xs',
+    },
+  }))
+  .add('Size XXS', () => ({
+    component: SpacingBottomComponent,
+    props: {
+      spacing: 'xxs',
+    },
+  }))
+  .add('Size XXXS', () => ({
```
Comment (RicardoNeves) [inline src/app/spacing/shared/directives/spacing-bottom/spacing-bottom.stories.ts]: @jmagalhaes-xing

**Pull request merged**